build.sbt

```diff
@@ -176,6 +176,9 @@ ThisBuild / mimaBinaryIssueFilters ++= Seq(
   ),
   ProblemFilters.exclude[ReversedMissingMethodProblem](
     "fs2.io.net.tls.TLSContext#Builder.insecureResource"
+  ),
+  ProblemFilters.exclude[DirectMissingMethodProblem]( // something funky in Scala 3.2.0 ...
+    "fs2.io.net.SocketGroupCompanionPlatform#AsyncSocketGroup.this"
   )
```

Comment:
Super weird, can reproduce just by updating to 3.2.0 on `main`. No idea ...

Reply:
I minimized it in https://github.com/armanbilge/sandbox/commit/d9453ac24ec96253b61431f0b30b529e353037b5. IDK what change in dotty caused it, but seems safe: seems like the private constructor was generating public bytecode prior to 3.2.0, but is now properly private. This is flagging up MiMa.

It doesn't affect anything public.